_search-plugins/cross-cluster-search.md

```diff
@@ -243,3 +243,56 @@ curl -XGET -k -u booksuser:password 'https://localhost:9250/opensearch-ccs-clust
   }]
 }
 ```
+
+## Sample bare metal/virtual machines setup
+
+If you are running OpenSearch on a bare metal server or using a virtual machine, you can run the same commands, specifying the IP (or domain) of the OpenSearch cluster.
+For example, in order to configure remote cluster for Cross-cluster search, find the IP of the remote node or domain of the remote cluster and run command:
+
+```json
+curl -k -XPUT -H 'Content-Type: application/json' -u 'admin:<custom-admin-password>' 'https://opensearch-domain-1:9200/_cluster/settings' -d '
+{
+  "persistent": {
+    "cluster.remote": {
+      "opensearch-ccs-cluster2": {
+        "seeds": ["opensearch-domain-2:9300"]
+      }
+    }
+  }
+}'
+```
+It is enough to point to only one of the node IPs on the remote cluster, as node discovery process will query all nodes in that cluster.
```

Comment:
```suggestion
It is sufficient to point to only one of the node IPs on the remote cluster because all nodes in the cluster will be queried as part of the node discovery process.
```